layers/+themes/theming/README.org

```diff
@@ -28,6 +28,30 @@ of the following form:
            (face2 attributes...)
            ...)
    ...)
+   
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
+#+begin_src lisp
+  ;; ~/.spacemacs
+  ;;...
+  (defun dotspacemacs/user-init ()
+  ;;...
+   (setq-default 
```

Comment:
1. `setq` is more suitable here
2. should have one more indent space (total indentation should be 2 spaces deeper than above), adjust lines below as necessary.